telemetry/specs/telemetry_report.mdk

```diff
@@ -424,47 +439,108 @@ The Destination Port is user configurable. The expectation is that the same
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+<-+
+      
+* Ver (4b): Version
+  : This specification defines **version 2.0**.
+
+* hw_id (6b): Hardware ID
+  : Identifies the hardware subsystem within the network device that generated
+    this report. For example, in a chassis with multiple linecards this could
+    identify a specific linecard, or a subsystem within a linecard. The hw_id
+    is unique within the scope of a Switch id.
+
+* Sequence Number (22b): Sequence Number
+  : Reflects the sequence of reports from a specific combination of
+    (Node id, hw_id) to a particular telemetry report destination. This
+    can be used to detect loss of telemetry reports before they reach their
+    intended destination.
 
-* Ver: Version
-  : This specification defines **version 1**.
+* Node ID (32b): Node ID
+  : The unique ID of a network device (such as routers, switches, NICs etc.) This is generally administratively assigned. Node IDs
+    must be unique within a management domain.
 
-* Length
-  : Indicates the length of the telemetry report header in multiples of
-    4 octets, including the variable optional metadata.
+* RepType (4b): Report Type
+  : Type of Telemetry report
+    * 0: INT v2.0 or later.
+    * 1: IOAM
+    * 2 – 15: Reserved
+
+* InType (4b): Inner Type
+  : Type of data embedded after Variable Optional Baseline & DS Metadata.
+    * 0: None
+    * 1: TLV
+    * 2: Domain Specific Extension Data
+    * 3: Ethernet
+    * 4: IPv4
+    * 5: IPv6
+    * 6-15: Reserved.
+
+* Report Length (8b): Report Length
+  : Indicates length of report header in multiple of 4-byte words, including the variable 
+    optional baseline metadata,  DS metadata and Truncated Packet or additional Domain 
+    Specific Extension Data.
+    
+* MD Length (8b): Metadata Length
+  : Indicates the length of Variable Optional Baseline Metadata and Variable Optional 
+    Domain Specific Metadata in 4-byte words.
+
+* D (1b): Dropped
+  : Indicates that at least one packet matching a drop watchlist was dropped.
 
-* NProt: Next Protocol
-  :
-    * 0: Ethernet
-    * 1: IPv4
-    * 2: IPv6
+* Q (1b): Congested Queue Association
+  : Indicates the presence of congestion on a monitored queue.
 
-* RepMdBits: Report Metadata Bits
+* F (1b): Tracked Flow Association
+  : Indicates that this telemetry report is for a tracked flow, i.e. the
+    packet matched a flow watchlist somewhere (in case of INT-MD, INT-MX or IOAM) or
+    locally (in case of INT-XD). The report might include INT-MD or IOAM
+    metadata beyond the inner ethernet header. Other telemetry reports are
+    likely to be received for the same tracked flow, from the same network
+    device and (in case of drop reports, INT-MX, INT-XD or path changes) from
+    other network devices.
+  
+* I (1b): Intermediate Report
+  : Indicates that a transit node sent this intermediate report for INT-MD.
```

Comment:
Can we provide a bit more context of this usecase?

Reply:
I suggested some text for INT 2.1

Reply:
Will add some comments in INT 2.1 spec.